### PR TITLE
[Exporter Refactor] Quantizable Conv Integer

### PR DESCRIPTION
**Pull request by dbogunowicz**
Adds:
- `convert_quantizable_conv_integer` transform

### CODE REVIEW

**Comment by corey-nm** [comment]
`GemmToMatMulInteger` (i.e. [_convert_quantizable_gemm_no_activations](https://github.com/neuralmagic/sparseml/blob/9778cec8684d3fcb683504eef88655d9a4607fd6/src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py#L884)) also uses the 

_add_quantized_conv_matmul_add_ops function. so should be easy to add after this is merged

File: src/sparseml/exporters/transforms/convert_quantizable_conv_integer.py
Comment (bfineran) [inline src/sparseml/exporters/transforms/convert_quantizable_conv_integer.py:32]: nit: add __all__ list

**Review by bfineran — COMMENTED** [review]
lgtm overall needs rebase

**Review by corey-nm — APPROVED** [review]
Let's get this rebased and merged in, this will unblock a couple of other passes 😁

**Comment by dbogunowicz** [comment]
on it

**Pull request merged**
